commands/Moderation/mute.js

```diff
@@ -0,0 +1,59 @@
+/*
+
+	To use this correctly, you will also need the unmute task located in
+	/tasks/unmute.js
+
+*/
+const { Command, Duration } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+			description: 'Mutes a mentioned user.',
+			usage: '[when:time] <member:user> [reason:string] [...]',
```

Comment:
member arg should use the member argument as you cannot mute someone who is not present in the server.